### PR TITLE
v12 release

### PR DESCRIPTION
**Pull request by fenekku**
- **dependencies: update for invenio-app-rdm v12**
- **release: v1.5.0**

### CODE REVIEW

File: invenio_cli/commands/requirements.py [hunk 1]
```diff
@@ -196,7 +196,10 @@ def check_pipenv_installed(cls):
     @classmethod
     def check_dev(cls):
         """Steps to check the development pre-requisites."""
-        if rdm_version()[0] >= 11:
+        if rdm_version()[0] >= 12:
```
Comment (fenekku) [inline invenio_cli/commands/requirements.py:199]: Honestly we should revisit this afterwards since this has a cost to maintain/gets out of sync.

Comment (utnapischtim) [reply invenio_cli/commands/requirements.py:199]: i thought the same. Since we thought about v12 is the next LTS we could skip the v9 support?

**Review by utnapischtim — APPROVED** [review]

File: invenio_cli/commands/requirements.py (hunk 1 above)
Comment (fenekku) [reply invenio_cli/commands/requirements.py:199]: We support v9 until Feb 2025, so we can remove it then.

**Comment by fenekku** [comment]
Once merged and released (I should be able to), I'll merge the docs too.

**Review by slint — APPROVED** [review]
:shipit:

**Pull request merged**